### PR TITLE
+yarnpkg.com

### PR DESCRIPTION
**Pull request by getneil**

### CODE REVIEW

File: projects/yarnpkg.com/package.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    yarnScript=$(cat scripts/bin/yarn)
+    replace="../.."
+    next=..
+    echo "${yarnScript/$replace/$next}" >> {{prefix}}/bin/yarn
+    chmod +x {{prefix}}/bin/yarn
+
+    cp {{prefix}}/bin/yarn {{prefix}}/bin/yarnpkg
+
+    if test "{{hw.platform}}" = "linux"; then
+      $SUDO apt-get --yes install libatomic1
```
Comment (getneil) [inline projects/yarnpkg.com/package.yml]: need help on this im trying to find the first source or how to package this.

Comment (jhheider) [reply projects/yarnpkg.com/package.yml]: it's _very_ low level. I believe we require it as a matter of course, so it shouldn't be necessary to install it.

**Comment by jhheider** [comment]
In theory, you might be able to install this using `npm install . --global --prefix={{prefix}}`.

**Comment by getneil** [comment]
> In theory, you might be able to install this using `npm install . --global --prefix={{prefix}}`.



> In theory, you might be able to install this using `npm install . --global --prefix={{prefix}}`.

that should have been the first thing i tried.

**Comment by jhheider** [comment]
It's not always obvious; I referred back to my work on `npmjs.com`. That's something that would be well-served by https://github.com/teaxyz/pantry.core/issues/142.

**Comment by getneil** [comment]
> npm install . --global --prefix={{prefix}}

unfortunately it still requires the libatomic1.

**Comment by jhheider** [comment]
That should be on all the images, unless something has changed. Almost everything requires it.

**Comment by getneil** [comment]
> That should be on all the images, unless something has changed. Almost everything requires it.

working for me now. i replaced my docker image w/ `ghcr.io/teaxyz/infuser`

**Comment by jhheider** [comment]
#teamwork

**Comment by jhheider** [comment]
and, I think, that might be on the top300.

**Pull request merged**